scrapers/FreakMobMedia.yml

```diff
@@ -0,0 +1,31 @@
+name: "freakmobmedia"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - freakmobmedia.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="bodywhite"]/div[@class="title"]//h3/text()
+      Details: //div[@class='description']/text()
```

Comment:
there are cases where 
- there is more than one text eg `http://www.freakmobmedia.com/sandra-luberc-gets-anal-creampie-from-bbc/`
- a `p` is inside the div instead of plain text `http://www.freakmobmedia.com/julianna-vega-and-queen-rogue-pov/`

In both cases i think a `concat` is needed and maybe a union of the text() and p/text() cases ?

Reply:
Found that there is a `//meta[@name="description"]/@content`. Find it everywhere. Easier to deal with.
Added a regex to remove stupid things